scanomatic/image_analysis/signal.py

```diff
@@ -727,7 +729,7 @@ def get_offset_quality(
     return q.sum()
 
 
-def _get_wave_length_and_errors(s, expected_spikes: int) -> float:
```

Comment:
Unused parameter

Reply:
I think these share an interface, hence why it has that argument, but @local-minimum can hopefully elucidate.

Reply:
I did a lot of yagni back then and probably used it while developing the algorithm to figure something out, but seems reasonable to remove now